test/Microsoft.ML.Tests/TrainerEstimators/CalibratorEstimators.cs

```diff
@@ -137,5 +137,101 @@ private sealed class CalibratorTestData
             }
         }
 
+        /// <summary>
+        /// Test to confirm calibrator estimators work with classes
+        /// where order of label and score columns are reversed, and
+        /// where 
+        /// </summary>
+        [Fact]
+        public void TestNonStandardCalibratorEstimatorClasses()
+        {
+            var mlContext = new MLContext(0);
+            // Store different possible variations of calibrator data classes.
```

Comment:
(pinned to text to allow for threaded discussion)

On issue #4700 I mention that Onnx tests are also necessary to test that Onnx conversion works also with a non-default Score name. A quick look makes me thing that it won't be necessary to change anything in the onnx conversion logic to work with the new `scoreColumnName` parameter, so simply adding a test will do.

The test would belong here:
https://github.com/dotnet/machinelearning/blob/709ec020b702fe69e0dcbf87fbcb538bea0af7b7/test/Microsoft.ML.Tests/OnnxConversionTest.cs#L316-L331

Reply:
I agree, and have added on to `PlattCalibratorOnnxConversionTest2` testing Onnx Conversion with a non-default Score column name for a Platt Calibrator. I will add other Onnx Conversion unit tests for other calibrators in a future PR.